### ISSUE TITLE
InstanceNormalizationPlugin cuDNN error Windows 10

### ISSUE THREAD

**Issue by ninono12345** [NVIDIA/TensorRT#3623]
## Description

Hello, I am working on a project to convert a pytorch tracking algorithm to TensorRT

I can successfully export the tracking algorithm to onnx then to TensorRT, but when I try to run inference on it, I get this error:
`[E] C:\_src\plugin\instanceNormalizationPlugin\instanceNormalizationPlugin.cu (335) - Cudnn Error in nvinfer1::plugin::InstanceNormalizationPlugin::enqueue: 14 (CUDNN_STATUS_VERSION_MISMATCH)`

the tracking module uses a couple of submodules, one of them is bb regressor, I exported it to onnx and we see that it contains InstanceNormalization:
![bb_regressor_sanitized](https://github.com/NVIDIA/cudnn-frontend/assets/116962708/be38f64a-6ce7-4282-b7d3-833daff7b9ad)

now, when I try to run inference on a pytorch docker, which is run on linux, the inference is ok, but I am using Windows 10 and for some reason it is not working

Keep in mind that I reinstalled the latest cuDNN 4 times. So I'm thinking it is a bug, or a Windows specific error, and it won't be fixed soon

NOW

I'm asking for your help, perhaps I can change the models code, so it doesn't use Instance Normalization, or edit the onnx graph?
Looking at the onnx graph we see that self.tower uses InstanceNormalization

model code:
`s

    class DenseBoxRegressor(nn.Module):
    def __init__(self, num_channels, project_filter=True):
        super().__init__()
        self.num_channels = num_channels
        self.project_filter = project_filter

        if self.project_filter:
            self.linear = nn.Linear(self.num_channels, self.num_channels)

        layers = []
        layers.extend(conv_layer(num_channels, num_channels))
        layers.extend(conv_layer(num_channels, num_channels))
        layers.extend(conv_layer(num_channels, num_channels))
        layers.extend(conv_layer(num_channels, num_channels))
        self.tower = nn.Sequential(*layers)

        self.bbreg_layer = nn.Conv2d(num_channels, 4, kernel_size=3, dilation=1, padding=1)

    def forward(self, feat, filter):
        nf, ns, c, h, w = feat.shape

        if self.project_filter:
            filter_proj = self.linear(filter.reshape(ns, c)).reshape(ns, c, 1, 1)
        else:
            filter_proj = filter

        attention = filter_layer.apply_filter(feat, filter_proj) # (nf, ns, h, w)
        feats_att = attention.unsqueeze(2)*feat # (nf, ns, c, h, w)

        feats_tower = self.tower(feats_att.reshape(-1, self.num_channels, feat.shape[-2], feat.shape[-1])) # (nf*ns, c, h, w)

        ltrb = torch.exp(self.bbreg_layer(feats_tower)).unsqueeze(0) # (nf*ns, 4, h, w)
        return ltrb`

Thank you for your help!

## Environment

**TensorRT Version**: 8.6.1

**NVIDIA GPU**: GTX 1660 Ti

**NVIDIA Driver Version**: 546.01

**CUDA Version**: 12.1 update 1

**CUDNN Version**: 8.9.7

Operating System: Windows 10

Python Version (if applicable): 3.10.13

PyTorch Version (if applicable): 2.1.2+cu121

## Relevant Files

**Model link**:

## Steps To Reproduce

**Commands or scripts**:

**Have you tried [the latest release](https://developer.nvidia.com/tensorrt)?**:

**Can this model run on other frameworks?** For example run ONNX model with ONNXRuntime (`polygraphy run <model.onnx> --onnxrt`):

**Comment by ninono12345** [comment]
@gs-olive perhaps you know what could be the problem?

**Comment by gs-olive** [comment]
Hi - I'm not very familiar with the `InstanceNormPlugin`, but a quick workaround might be to set the `tensorrt.OnnxParserFlag.NATIVE_INSTANCENORM` flag via `onnx_parser.set_flag(...)` - @pranavm-nvidia do you have any suggestions?

**Comment by zerollzeng** [comment]
> [E] C:\_src\plugin\instanceNormalizationPlugin\instanceNormalizationPlugin.cu (335) - Cudnn Error in nvinfer1::plugin::InstanceNormalizationPlugin::enqueue: 14 (CUDNN_STATUS_VERSION_MISMATCH)

Looks like it's caused by the cuDNN version, which version you are installed in your system?

**Comment by ninono12345** [comment]
@zerollzeng I have the latest installed 8.9.7 The this is that on the pytorch docker this can run, but in the docker the cudnn version is the same. So I'm thinking maybe it's a windows problem, or perhaps something is missing...
@gs-olive is this possible to make with trtexec from command prompt?

**Comment by zerollzeng** [comment]
A possible is you might also install other framework that bundle cuda, like pytorch. which contains a cuDNN.

**Comment by ninono12345** [comment]
@zerollzeng can you suggest anything?

**Comment by zerollzeng** [comment]
I don't have much suggestions since I almost know nothing about Windows :-(

**Comment by ninono12345** [comment]
@gs-olive, thank you for you answer, and OMG it worked :D . Somehow I had a feeling that you would know. I feel really stupid, because the answer was inside the developers guide all this time, I just got lost in there, and missed it.

**Comment by zerollzeng** [comment]
Problem solved? Can we close this issue?

**Comment by ninono12345** [comment]
yes

**Comment by DYM-ing** [comment]
Hello, I met the same problem, how did you solve it？ Could you tell me？

**Comment by ninono12345** [comment]
> Hi - I'm not very familiar with the `InstanceNormPlugin`, but a quick workaround might be to set the `tensorrt.OnnxParserFlag.NATIVE_INSTANCENORM` flag via `onnx_parser.set_flag(...)` - [@pranavm-nvidia](https://github.com/pranavm-nvidia) do you have any suggestions?

Here. Look at tensorrt documentation and lookup instancenorm
